roles/undercloud/tasks/main.yaml

```diff
@@ -17,6 +17,8 @@
   become_user: stack
 
 - name: Install undercloud (this will take a while!)
+  environment: 
+        DIB_INSTALLTYPE_puppet_modules: package
```

Comment:
This needs to pick up the value set in the hosts file.

```
  environment
    DIB_INSTALLTYPE_puppet_modules: {{ dib_install_type_puppet_modules }}
```
(also, the indentation is reduced a bit here)